### PR TITLE
Fixes

### LINKED ISSUE #33: [enhancement] Add localization support
Would be nice to have the CapitalBe localizated, through the use of catkeys.  

I can contribute with the spanish translation.  May I help translating the "CapitalEnglish" file available in src/R5LanguageFiles/ ?

### PR DESCRIPTION
**Pull request by bitigchi**
Fixes #33.

### CODE REVIEW

**Comment by bitigchi** [comment]
@humdingerb, along with the others could you please check the catkeys commit? I am unable to build with reference errors. I'm sure I'm missing something obvious.

File: Makefile [hunk 1]
```diff
@@ -119,7 +119,7 @@ RSRCS = \
 #	- 	if your library does not follow the standard library naming scheme,
 #		you need to specify the path to the library and it's name.
 #		(e.g. for mylib.a, specify "mylib.a" or "path/mylib.a")
-LIBS = be tracker translation sqlite3 columnlistview $(STDCPPLIBS)
+LIBS = be tracker localestub sqlite3 columnlistview $(STDCPPLIBS)
```
Comment (humdingerb) [inline Makefile:122]: Still needs the "translation" kit for GetBitmap()

```suggestion
LIBS = be tracker localestub translation sqlite3 columnlistview $(STDCPPLIBS)
```

File: src/MainWindow.cpp [hunk 2]
```diff
@@ -31,219 +30,205 @@
...
+#define B_TRANSLATION_CONTEXT "MainWindow"
+
+
+// #define TEST
+// #define NOSAVE
 
 // Internal definition of the Haiku services daemon
 #define B_SERVICES_DAEMON_RESTART 'SDRS'
 
-Language *gCurrentLanguage = NULL;
```
Comment (humdingerb) [inline src/MainWindow.cpp:42]: The init of gCurrentLanguage is needed.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by bitigchi** [comment]
Build is fixed, but getting a segmentation error... Needle in a haystack.

**Comment by korli** [comment]
Please rebase on master to get the CI build testing.

**Comment by korli** [comment]
Please reference #33 if that's supposed to be fixed with this PR.

**Comment by bitigchi** [comment]
> Please reference #33 if that's supposed to be fixed with this PR.

Done.

File: .clang-format [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+PenaltyBreakBeforeFirstCallParameter: 19
+PenaltyBreakComment: 300
+PenaltyBreakFirstLessLess: 120
+PenaltyBreakOpenParenthesis: 0
+PenaltyBreakString: 1000
+PenaltyBreakTemplateDeclaration: 10
+PenaltyExcessCharacter: 1000000
+PenaltyIndentedWhitespace: 0
+PenaltyReturnTypeOnItsOwnLine: 60
+PointerAlignment: Right
```
Comment (korli) [inline .clang-format:168]: Have you considerd changing PointerAlignment to Left ?

Comment (bitigchi) [reply .clang-format:168]: Why not. This is a WIP, so anything that will make it closer to Haiku coding guidelines is welcome. Feel free to indicate more issues.

Comment (humdingerb) [reply .clang-format:168]: I find that the defaults coming with haiku-format do a good job. Besides with .h files and e.g. layout building code.

Comment (bitigchi) [reply .clang-format]: Done.

Comment (korli) [reply .clang-format]: The files need a new format pass after this change, don't they?

Comment (bitigchi) [reply .clang-format]: You might have a point there.

**Comment by korli** [comment]
Merged manually in master. Thanks!

**Comment by korli** [comment]
@bitigchi just saw that you added Setttings/SettingsWindows in the commit "Change pointer alignment to left". That's unfortunate.  They weren't added to the build anyway.

**Comment by bitigchi** [comment]
Ah I see. That was an initial refactor but never got around to finish it.

**Pull request closed without merging**